src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -1289,4 +1290,1373 @@ void EIRPlantLoopHeatPump::oneTimeInit(EnergyPlusData &state)
+
+        auto objType = (DataLoopNode::ConnectionObjectType)getEnumerationValue(BranchNodeConnections::ConnectionObjectTypeNamesUC,
+                                                                               UtilityRoutines::MakeUPPERCase(cCurrentModuleObject));
+        int numPLHP = state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, cCurrentModuleObject);
+        if (numPLHP > 0) {
+            auto const instances = state.dataInputProcessing->inputProcessor->epJSON.find(cCurrentModuleObject);
+            if (instances == state.dataInputProcessing->inputProcessor->epJSON.end()) {
+                // Cannot imagine how you would have numPLHP > 0 and yet the instances is empty
+                // this would indicate a major problem in the input processor, not a problem here
+                // I'll still catch this with errorsFound but I cannot make a unit test for it so excluding the line from coverage
+                ShowSevereError(state,                                                                     // LCOV_EXCL_LINE
+                                "EIR PLFFHP: Somehow getNumObjectsFound was > 0 but epJSON.find found 0"); // LCOV_EXCL_LINE
+                errorsFound = true;                                                                        // LCOV_EXCL_LINE
+            }
+            auto &instancesValue = instances.value();
+            for (auto instance = instancesValue.begin(); instance != instancesValue.end(); ++instance) {
+                auto const &fields = instance.value();
+                auto const &thisObjectName = instance.key();
+                state.dataInputProcessing->inputProcessor->markObjectAsUsed(cCurrentModuleObject, thisObjectName);
+
+                EIRFuelFiredHeatPump thisPLHP;
+                thisPLHP.EIRHPType = classToInput.thisType;
+                // A1-A3
+                thisPLHP.name = UtilityRoutines::MakeUPPERCase(thisObjectName);
+                std::string loadSideInletNodeName = UtilityRoutines::MakeUPPERCase(fields.at("water_inlet_node_name").get<std::string>());
+                std::string loadSideOutletNodeName = UtilityRoutines::MakeUPPERCase(fields.at("water_outlet_node_name").get<std::string>());
+                // Implicit
+                std::string condenserType = "AIRSOURCE"; // UtilityRoutines::MakeUPPERCase(fields.at("condenser_type").get<std::string>());
+                // A4
+                std::string sourceSideInletNodeName = UtilityRoutines::MakeUPPERCase(fields.at("air_source_node_name").get<std::string>());
+                std::string sourceSideOutletNodeName =
+                    "DUMMY_CONDENSER"; // UtilityRoutines::MakeUPPERCase(fields.at("source_side_outlet_node_name").get<std::string>());
+                srand(time(NULL));
+                for (int i = 1; i <= 3; ++i) {
+                    sourceSideOutletNodeName += ("_" + std::to_string(rand()));
+                }
```

Comment:
What is happening here?

Reply:
Changed to the following in earlier commits pushed: 
```
            srand(time(NULL));
            std::string sourceSideOutletNodeName = format("DUMMY_CONDENSER_{}_{}", rand(), rand());
```